### PR TITLE
Add systemd.timer to take backups of CKE-managed etcd

### PR DESCRIPTION
**Pull request by ymmt2005**

### CODE REVIEW

File: debian/lib/systemd/system/backup-cke-etcd.timer [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=run ckecli etcd local-backup every hour
+After=cke.service
+
+[Timer]
+OnBootSec=10min
+OnUnitActiveSec=1h
+RandomizeDelaySec=5min
```
Comment (morimoto-cybozu) [inline debian/lib/systemd/system/backup-cke-etcd.timer:8]:
```suggestion
RandomizedDelaySec=5min
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

**Review by morimoto-cybozu — DISMISSED** [review]
LGTM

**Comment by ymmt2005** [comment]
PTAL @morimoto-cybozu

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**